timur/config.ru

```diff
@@ -9,6 +9,8 @@ require_relative 'lib/server'
 
 Timur.instance.configure(YAML.load(File.read('config.yml')))
 
+# Used primarily by development servers.  These are served via apache in production.
+use Rack::Static, urls: ['/css', '/fonts', '/images', '/js'], root: 'public'
```

Comment:
Lets Apache serve the static files, so they aren't processed by Etna Auth middleware.

Reply:
:+1: